saga/fx/put.ts

```diff
@@ -0,0 +1,12 @@
+import { action } from "../../instructions.ts";
+
+import { ActionContext } from "../context.ts";
+import type { Action } from "../types.ts";
+
+export function put(act: Action) {
+  return action(function* (resolve) {
+    const { input } = yield* ActionContext;
+    input.send(act);
+    resolve(true);
+  });
+}
```

Comment:
Is there a need to make this run in its own hermetically sealed `Frame` by using `action()`? You could also just write it as a simple generator function:

```ts
export function* put(act: Action) {
  const { input } = yield* ActionContext;
  yield*  input.send(act);
  return true;
}